buyer-api/src/utils/wibson-lib/s3/index.js

```diff
@@ -36,7 +36,7 @@ const getS3Objects = async (namespace, justSnippets = false) => {
 // //
 
 const countObjects = async (dataOrder, type) => {
-  const namespace = `${prefix}/${dataOrder.address}/${type}/`;
+  const namespace = `${prefix}/${dataOrder.orderAddress}/${type}/`;
```

Comment:
@Meketh @Agus since `listObjects` and `getObject` both access `dataOrder.address` I would expect that `countObjects` also access the same field.
I'm sure the problem that forced this PR came from the changes in: https://github.com/wibsonorg/buyer-sdk/blob/5c07f36d423ad79ddab316a412d874d5696d4a9d/buyer-api/src/facades/getOrdersFacade.js#L91-L98 `order` has an `address` field and `dataOrder` has an `orderAddress`.
A good way to reduce the possibility of having this issue again is to use an address string instead of a dataOrder object (which means removing the knowledge this library has about a dataOrder).
What do you think? We can create an issue about it and tackle it when we finish with the urgent tasks.

Reply:
That's right, maybe it's just a matter of sending an "address", regardless its parent object (dataOrder, order, etc).